tests/testthat/test-5transformCounts.R

```diff
@@ -166,25 +157,16 @@ test_that("transformCounts", {
         expect_equal(as.matrix(assays(mia::transformCounts(tse, method = "relabundance"))$relabundance),
                      as.matrix(assays(mia::transformSamples(tse, method = "relabundance"))$relabundance))
         
-        # Tests transformFeatures, calculates clr-transformation. Should be equal.
-        # Calculates relative abundance table
-        relative <- expect_warning(assay(mia::transformFeatures(tse, method = "relabundance", 
-                                                                pseudocount = pseudonumber),
-                                         "relabundance"),
-                                   "Relative abundances vary")
-        expect_equal(
-            as.matrix(assays(mia::transformFeatures(tse, method = "clr", pseudocount = pseudonumber))$clr),
-            t(apply(as.matrix(t(relative)), 2, FUN=function(x){
-                log(x) - mean(log(x))
-            })))
+        # Tests transformFeatures, tries to calculate clr. Should be an error.
+        expect_error(mia::transformFeatures(tse, method = "clr"))
 
         ############################# NAMES ####################################
         # Tests that samples have correct names
         expect_equal(colnames(assays(transformCounts(tse, method = "clr", pseudocount = 1))$clr),
                      colnames(assays(tse)$counts))
 
         # Tests that otus have correct names
-        expect_equal(rownames(assays(transformFeatures(tse, method = "hellinger", pseudocount = 1000))$hellinger),
+        expect_equal(rownames(assays(transformFeatures(tse, method = "log10", pseudocount = 1000))$log10),
```

Comment:
Why not test both `hellinger` and `log10`?

Reply:
`transformFeatures `does not provide `hellinger` anymore. It provides just `log10`,` pa,` and `z`